src/commands/create/utils.ts

```diff
@@ -99,6 +99,30 @@ export const setupTemplate = async (
   packageManager: PackageManagerType
 ) => {
   Logger.info(`\nSetting up template in ${chalk.magentaBright(folderLocation)}...`);
+  if (packageManager === "forge") {
+    const spinner = ora("Initializing foundry project...").start();
+    const manager = packageManagers[packageManager];
```

Comment:
Hm, I'm not sure we need to add forge to package managers, since it's not really a package manager for npm packages and also from my understanding you don't really have a choice when it comes to foundry.
So I would recommend executing forge commands directly.

Reply:
@JackHamer09 any immediate recommendations here. We could introduce a `FoundryCmd` type for `forge` and maybe future `cast` usage and follow the same file structure as `packageManager.ts`. 

That is, something like:

```
export type FoundryCmd = "forge" | "cast";

interface ForgeMethods {
  install(packages?: string): string;
  build(script: string): string;
  uninstall(packages?: string): string;
  isInstalled(): Promise<boolean>;
  init?(gitUrl: string, folderLocation: string): Promise<boolean>;
}

export const forgeCmds: Record<FoundryCmd, ForgeMethods> = {
  forge: {
     ...
  }
 };
```

Open to other suggestions / recommendations.